README.md

```diff
@@ -1,9 +1,116 @@
+1. Complete the [prerequisites](#install-prerequisites) steps.
+2. Set the ```token``` environment variable equal to the agent access token copied from the Lacework Console.
+   ```
+   $ export token=<copied_agent_access_token>
+   ```
+3. Use ```sed``` to replace ```$1``` with a valid agent access token in the ```install.sh``` file.
+   ```
+   $ sed -i.bak "s/ARG1=\$1/ARG1=${token}/g" /tmp/install.sh
+   ```
+4. Run the install.sh script to install the agent by following the instructions in [Run the Lacework Agent Installation Script](https://support.lacework.com/hc/en-us/articles/360005321273).
+
+## Install using the Lacework Agent Chef Recipe
+1. Complete the [prerequisites](#install-prerequisites) steps.
+2. Set the ```token``` environment variable equal to the agent access token copied from the Lacework Console.
+   ```
+   $ export token=<copied_agent_access_token>
+   ```
+3. Use ```sed``` to replace ```$1``` with a valid agent access token in the ```install.sh``` file.
+   ```
+   $ sed -i.bak "s/ARG1=\$1/ARG1=${token}/g" /tmp/install.sh
+   ```
+4. Unzip ```chef.tar.gz``` into a temporary directory such as ```/tmp/chef```.
+   ```
+   $ tar -xzf chef.tar.gz
+   ```
+5. Copy the ```install.sh``` file with the updated token to the appropriate directory.
+   ```
+   $ cp install.sh /tmp/chef/datacollector/files/default
+   ```
+6. Install the agent by following the instructions in [Install with Chef](https://support.lacework.com/hc/en-us/articles/360005321413).
 
-Latest Lacework Documentation are available <a href="https://support.lacework.com/hc/en-us" target="_blank">**here**</a>.
+## Install using Daemonset Deployment files for Kubernetes
+1. Complete the [prerequisites](#install-prerequisites) steps.
+2. Set the ```token``` environment variable equal to the agent access token copied from the Lacework Console.
+   ```
+   $ export token=<copied_agent_access_token>
+   ```
+3. Use ```sed``` to replace ```LaceworkAccessToken``` with a valid agent access token in the ```lacework-cfg-k8s.yaml``` file.
+   ```
+   $ sed -i.bak "s/\${LaceworkAccessToken}/${token}/g" lacework-cfg-k8s.yaml
+   ```
+4. Install the agent on Kubernetes by following the instructions in [Deploy on Kubernetes](https://support.lacework.com/hc/en-us/articles/360005263034).
+   On the **Releases** page of this site, find the appropriate ```docker pull``` command for a specific release under the **Lacework Agent Docker image** heading.
+
+## Install using Docker Swarm Deployment Files
+1. Complete the [prerequisites](#install-prerequisites) steps.
+2. Set the ```token``` environment variable equal to the agent token copied from the Lacework Console.
+   ```
+   $ export token=<copied_agent_access_token>
+   ```
+3. Use ```sed``` to  replace ```LaceworkAccessToken``` with the valid agent access token in the ```docker-compose.yml``` and ```docker-compose-v3.yml``` files.
+   ```
+   $ sed -i.-i.bak "s/\${LaceworkAccessToken}/${token}/g" docker-compose.yml
```

Comment:
Please omit -i. from both commands. this should say something like -
```sed -i.bak "s/\${LaceworkAccessToken}/${token}/g" docker-compose.yml```

Reply:
Good catch. fixed.